website/content/docs/discovery/dns.mdx

```diff
@@ -353,25 +358,49 @@ $ echo -n "20010db800010002cafe000000001337" | perl -ne 'printf join(":", unpack
 
 By default, all service lookups use the `default` namespace
 within the partition and datacenter of the Consul agent that received the DNS query.
-To lookup services in another namespace, partition, and/or datacenter,
+To lookup services in another namespace, partition, datacenter, or cluster peer,
 use the [canonical format](#canonical-format).
 
 Consul server agents are in the `default` partition.
 If DNS queries are addressed to Consul server agents,
 service lookups to non-`default` partitions must explicitly specify
 the partition of the target service.
 
-To lookup services imported from a cluster peer,
-refer to [service virtual IP lookups for Consul Enterprise](#service-virtual-ip-lookups-for-consul-enterprise) instead.
+To lookup a mesh-participating service that's imported from a cluster peer,
+refer to [service virtual IP lookup for Consul Enterprise](#service-virtual-ip-lookups-for-consul-enterprise) instead.
 
 #### Canonical format
 
-Use the following query format to specify namespace, partition, and/or datacenter
+Use the following query formats to specify namespace, partition, datacenter, or peer
 for `.service`, `.connect`, `.virtual`, and `.ingress` service lookup types.
-All three fields (`namespace`, `partition`, `datacenter`) are optional.
-```text
-[<tag>.]<service>.service[.<namespace>.ns][.<partition>.ap][.<datacenter>.dc]<domain>
-```
+In each format, all fields in square brackets are optional.
+
+- Lookup a service, optionally in a cluster peer:
+
+  ```text
+  [<tag>.]<service>.service[.<namespace>.ns][.<peer>.peer][.<partition>.ap].<domain>
```

Comment:
We don't currently support this format with `.service.p1.peer` in 1.14.0, so we'll need to remove anything referring to it. I believe we'll only be able to use the new `.peer` syntax with `virtual` entries for the time-being.

Reply:
Sounds good - I'll move that stuff out into another PR ...

... once the accuracy of some of the other content is reviewed (so I can batch all my changes).

I'm particularly not sure about the changes I made to the `### Service Virtual IP Lookups` section.

Reply:
Note to self: `.peer` support varies by service lookup type and version.

`.virtual` has supported as of 1.14.0.
`.service` and `.node`  have support [as of 1.14.2](https://github.com/hashicorp/consul/blob/main/CHANGELOG.md#1142-november-30-2022).

`.ingress` and `.connect` lack support.